stable/rabbitmq-ha/templates/statefulset.yaml

```diff
@@ -285,6 +286,10 @@ spec:
                     app: {{ template "rabbitmq-ha.name" . }}
                     release: {{ .Release.Name }}
       {{- end }}
+    {{- with .Values.podAffinity }}
```

Comment:
I don't like the idea of using `podAffinity` to be able to set `podAntiAffinity` as mentioned in the README. Either add a label here of `podAffinity` to disallow that or change this setting to be just `affinity` which will take precedence over soft or hard anti-affinity. This would also allow it to be used for `nodeAffinity`.